libyul/YulStack.h

```diff
@@ -65,7 +65,7 @@ struct MachineAssemblyObject
 class YulStack: public langutil::CharStreamProvider
 {
 public:
-	enum class Language { Yul, Assembly, StrictAssembly };
+	enum class Language { StrictAssembly };
```

Comment:
I have opted for `StrictAssembly` here but in the end it doesn't really matter, I guess